src/main/resources/plugin-additional-permissions.yml

```diff
@@ -0,0 +1,21 @@
+cluster_permissions:
```

Comment:
How will we use this file? Is it related to https://github.com/opensearch-project/opensearch-plugins/issues/238?